spec/services/authn_context_resolver_spec.rb

```diff
@@ -427,4 +427,281 @@
       end
     end
   end
+
+  context 'when resolving semantic acr_values' do
+    before do
+      allow(IdentityConfig.store).
+        to receive(:feature_valid_authn_contexts_semantic_enabled).
+        and_return(true)
+      allow_any_instance_of(ServiceProvider).
+        to receive(:semantic_authn_contexts_allowed?).
+        and_return(true)
+
+      stub_const(
```

Comment:
[question] hm, why does this need to be stubbed? shouldn't setting the config do this?

Reply:
Unfortunately, no. Since it is a const, it is evaluated too early to be effected by the flag mocking happening here so we have to stub the const.

I'm also mocking the flags since `VALID_AUTHN_CONTEXTS` isn't the only place where those are used.